mocks/firestore.js

```diff
@@ -65,6 +65,10 @@ class FakeFirestore {
     };
   }
 
+  settings(settings) {
```

Comment:
Could we add a `mockSettings` call to this? On the off chance someone wants to test that their settings are called correctly.

```
const mockSettings = jest.fn();
...
settings() {
  mockSettings(...arguments);
  return;
}
```

Reply:
Sure thing. no problem